src/qt/bitcoingui.cpp

```diff
@@ -362,7 +362,7 @@ void BitcoinGUI::createActions()
 
 #ifdef ENABLE_WALLET
     QSettings settings;
-    if (!fLiteMode && settings.value("fShowMasternodesTab").toBool()) {
+    if (settings.value("fShowMasternodesTab").toBool()) {
```

Comment:
Need similar changes in `src/qt/walletview.cpp`

Reply:
51862ab695d52a655896ac29ef4ed884eafe904d